hd/etc/css.txt

```diff
@@ -35,7 +35,7 @@
     %end;
   %end;
 %end;
-<link rel="stylesheet" href="%etc_prefix;css/css.css">
+<link rel="stylesheet" href="%etc_prefix;css/css.css?hash=%apply;hash("css/css.css")">
```

Comment:
Did you try to use the integrity field? 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement/integrity

I believe that browsers will download again the file if the integrity field does not match the hash of the file.